R/utils/check.r

```diff
@@ -0,0 +1,53 @@
+appendGenes <- function(mouseGenesVector, GSEMatrix)
```

Comment:
I would change the arguments to something more generic:

`appendGenes <- function(geneVec, mat)`

Reply:
See commit 72b0074